build-system/tasks/jsify-css.js

```diff
@@ -45,6 +45,9 @@ cssnano = cssnano({
   mergeIdents: true,
   reduceIdents: false,
   zindex: false,
+  svgo: {
+    encode: true,
```

Comment:
Ah yes, it's here: https://github.com/svg/svgo/blob/master/lib/svgo/coa.js#L288
Looks like they pass the input string into svgo, which outputs an unencoded svg, which is then encoded.

I'll still update our sources, it's just good to know this won't break unexpectedly if someone pastes an svg with URL0encoded characters in it.